modules/Hoot/managers/api.js

```diff
@@ -574,6 +574,31 @@ export default class API {
             .then( resp => resp.data );
     }
 
+    resolveAllConflicts(mapId) {
+        const params = {
+            path: '/job/review/resolveall',
+            method: 'PUT',
+            data: {mapId: mapId}
+        };
+
+        return this.request( params )
+        .then( resp => {
+            if (resp.status !== 200) {
+                return Promise.reject({
```

Comment:
I think you can just do the alert stuff, no need for the internal promise.reject